src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/HttpStreamFormatter.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using CoreWCF.Channels;
+using CoreWCF.Description;
+
+namespace CoreWCF.Dispatcher
+{
+    internal class HttpStreamFormatter : IDispatchMessageFormatter, IClientMessageFormatter
+    {
+        readonly string _contractName;
+        readonly string _contractNs;
```

Comment:
Missing accessibility modifier in multiple places in this file

Reply:
This was addressed in 7bf2248d44d5d53bedc75e6fa257c08047456f4e.